misc/slack-notification/slack.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+MESSAGE="Successfully finished running '${ENV0_DEPLOYMENT_TYPE}' deployment on"
+
+if [[ -z "${ENV0_DEPLOYMENT_REVISION}" ]]; then
+  MESSAGE="${MESSAGE} repository's default branch.\n"
+else
+  MESSAGE="${MESSAGE} revision '${ENV0_DEPLOYMENT_REVISION}'.\n"
+fi
+
+if [[ -z "${ENV0_REVIEWER_NAME}" ]]; then
+  MESSAGE="${MESSAGE} Automatically approved."
+else
+  MESSAGE="${MESSAGE} Approved by: ${ENV0_REVIEWER_NAME}(${ENV0_REVIEWER_EMAIL})"
+fi
+
+PAYLOAD="{\"channel\": \"#${SLACK_CHANNEL}\", \"username\": \"webhookbot\", \"text\": \"${MESSAGE}\", \"icon_emoji\": \":env0:\"}"
```

Comment:
so that won't work
it work for us since we have `env0` emoji
use https://uploads-ssl.webflow.com/5ceab5395d0f478e169de7c0/5f21eb54b57c7a8559c68ff7_Env0-Color%20Icon%403x.png

Reply:
thought this repo was for internal use, will change.
85c59bc452da4dc5a0fb964b93344e4f30e14556

Reply:
we want to send that to the customer that asked us for example